hw2/05-form.js

```diff
@@ -0,0 +1,32 @@
+const registrationForm = document.querySelector('.registration-form');
+
+const logForm = function captureAndLogRegistrationForm(event) {
+  const {
+    fullName, email, registrationStatus, coursesTaken, anythingElse,
```

Comment:
Just wanted to say this was really nice! I didn't think about destructuring from the event.target object!